### DISCUSSION TITLE
Datos faltantes en dataset trabajo final

### DISCUSSION THREAD

**Discussion by moralesluis26** [rramosp/ai4eng.v1#175] [00 - General]
Saludos profesor y monitor.

Tengo una duda respecto a los datos faltantes que debe tener mi dataset.
Según entinedo, de los datos disponibles de mi data set, el 5% deben ser faltantes y deben estar distribuidos en por lo menos 3 columnas.
¿Estaría bien si tomo varias filas de forma aleatoria y reemplazo sus valores por faltantes (así como muestro en la figura)?
Agradezco la atención
![imagen](https://user-images.githubusercontent.com/46792756/181420742-a922ce0d-6f9c-4900-97d0-68039410560c.png)

**Comment by Felipe-RA** [comment]
Saludod moralesluis26

Es un buen primer acercamiento, pero te recomiendo no convertir toda la fila a nan, sino algunos valores de cada fila

El razonamiento de esto es que no tiene mucho sentido tener una fila sin ningún dato anotado, pues sería en realidad indicador de un problema de procesamiento de los datos y no de falta natural de información.

Te pongo un ejemplo con los coches:

Es más natural que a muchas filas les falte, por ejemplo,  drivewheel y enginelocation (puede ser que la persona que anotó esas filas no lo tuvo en cuenta, o que esas filas provengan de otro experimento en el que no se tenían en cuenta esas dos características)

En contraste, tener filas faltantes completamente es sospechoso

**Reply by moralesluis26** [reply]
Saludos Felipe-RA.

Entendido. Con ese orden de ideas, decidí implementar en mi df la función `replace()` en la cual podría eliminar ciertos datos específico tal como se evidencia en la imagen que adjunto. ¿Haciendo esto estaría mejor? o qué me podría recomendar para hacer más natural la falta de datos. 
![imagen](https://user-images.githubusercontent.com/46792756/181666843-57189ddb-b121-4eee-ad9a-f3dae964f39e.png)

Agradezco de antemano.

**Comment by Felipe-RA** [comment]
Creo que con eso vas bien!
